examples/telegram-bot/src/TelegramBot.ts

```diff
@@ -54,9 +54,12 @@ export class TelegramBot {
       // Set up tools 
       // Get balance of agent account
       const checkBalance = this.agent.getNativeBalanceTool();
+      // transfer native token to another address on specific chain 
```

Comment:
Please keep the comment consistent
```suggestion
      // Transfer native tokens to a recipient address on a specific chain.
```